prog/vnet/subnet_nexus.rb

```diff
@@ -154,12 +161,11 @@ def gen_reqid
       nap 5
     end
 
-    decr_destroy
```

Comment:
I believe there's no need to move this `decr_destroy`. We also decrement the semaphore first in other progs.